From 3df92fc73e68d61959738d59be557427af917dd0 Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Thu, 12 Jul 2018 15:29:25 +0800
Subject: [PATCH 267/706] drm: hdp: Update Lane link rate from 1.6Gb/s to
 2.7Gb/s

This patch updates Lane link rate from 1.6Gb/s to 2.7Gb/s.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
(cherry picked from commit a6129d3228a04887d65058e5d0510ef81e50c39d)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdp.c b/drivers/gpu/drm/imx/hdp/imx-hdp.c
index 0ecc42940bf4..42348e4cf33d 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdp.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdp.c
@@ -976,7 +976,7 @@ static int imx_hdp_imx_bind(struct device *dev, struct device *master,
 	}
 #endif
 
-	hdp->link_rate = AFE_LINK_RATE_1_6;
+	hdp->link_rate = AFE_LINK_RATE_2_7;
 
 	hdp->dual_mode = false;
 
-- 
2.17.1

